src/plugins/syscalls/syscalls.cpp

```diff
@@ -128,10 +130,99 @@ static event_response_t cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
     return 0;
 }
 
+static event_response_t win_cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
+    int i,j;
+
+    syscall_wrapper_t *wrapper = (syscall_wrapper_t*)info->trap->data;
+    syscalls *s = wrapper->sc;
+
+    vmi_instance_t vmi = drakvuf_lock_and_get_vmi(drakvuf);
+
+    uint8_t reg_size = vmi_get_address_width(vmi);
```

Comment:
I think it would be enough to get this once in the plugin constructor and cache it in the class itself. No point doing this for every callback.